templates/event_cancellation.html

```diff
@@ -0,0 +1,67 @@
+                <div style="box-sizing: border-box;">
+                    <h5 class="greeting" style="box-sizing: border-box;margin-top: 0;margin-bottom: .5rem;font-family: inherit;font-weight: 500;line-height: 31px;color: inherit;font-size: 24px;padding-top: 63px;">
+                        <Strong>Cancelled Event!</strong>
+                    </h5>
+                </div>
+                <div class="rectangle-3" style="box-sizing: border-box;margin: 45px 105px;">
+                    <p class="invite" style="box-sizing: border-box;margin-top: 0;margin-bottom: 0;orphans: 3;widows: 3;color: #6d6d6d;line-height: 21px;font-family: 'DINPro', 'Roboto', sans-serif !important; text-align: unset; width: max-content;">
+                        <strong> {{ event_title }} </strong> event at <strong> {{ room_name }} </strong> was just cancelled <br/> 
+                        and removed from your calendar with this note:<br/>
+                        "Cancelling because of missed checkin."
+                    </p>
+                </div>
+                <div style="box-sizing: border-box;">
+                    <hr class="line" style="box-sizing: content-box;height: 0;overflow: visible;margin-top: 70px;margin-bottom: 1rem;border: 0;border-top: 1px solid rgba(0,0,0,.1);">
+                </div>
+            </div>
+            <div class="rectangle-2" style="box-sizing: border-box;height: 130px;width: 610px;border-radius: 0 0 4px 4px;">
+                <div class="contact" style="box-sizing: border-box;height: 54.32px;width: 490px;font-family: 'DINPro', 'Roboto', sans-serif, sans-serif;color: #000000;font-size: 16px;line-height: 27px;padding-top: 39px;padding-bottom: 39px;margin: auto;text-align: center;">
+                    <p style="box-sizing: border-box;margin-top: 0;margin-bottom: 1rem;orphans: 3;widows: 3;">
+                        Have a question?
+                        <br style="box-sizing: border-box;"> <a href="#" style="box-sizing: border-box;color: #007bff;text-decoration: underline;background-color: transparent;-webkit-text-decoration-skip: objects;">
+                            support@converge.com</a>
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="subcription" style="box-sizing: border-box;height: 19.16px;width: 490px;color: #000111;font-family: 'DINPro', 'Roboto', sans-serif, sans-serif;font-size: 16px;line-height: 19px;text-align: center;margin: auto;padding-top: 15px;">
+            <p style="box-sizing: border-box;margin-top: 0;margin-bottom: 1rem;orphans: 3;widows: 3;">You can change your<span style="box-sizing: border-box;"><a
+                        id="link" href="https://converge-test.andela.com/preference" style="box-sizing: border-box;color: #999999;text-decoration: underline;background-color: transparent;-webkit-text-decoration-skip: objects;">
+                        <strong>subcription settings </strong></a></span> anytime.</p>
```

Comment:
Kindly correct the spelling of 'subscription' so it displays correctly when the email is sent